library/std/Cargo.toml

```diff
@@ -25,7 +25,7 @@ hashbrown = { version = "0.14", default-features = false, features = ['rustc-dep
 std_detect = { path = "../stdarch/crates/std_detect", default-features = false, features = ['rustc-dep-of-std'] }
 
 # Dependencies of the `backtrace` crate
-addr2line = { version = "0.19.0", optional = true, default-features = false }
+addr2line = { version = "0.20.0", optional = true, default-features = false }
 rustc-demangle = { version = "0.1.21", features = ['rustc-dep-of-std'] }
 miniz_oxide = { version = "0.6.0", optional = true, default-features = false, public = false }
```

Comment:
This was bumped in `backtrace` submodule too.

Reply:
yes, but without bumping `addr2line` here, `x b` got errors:

```console
~/rust ❯❯❯ x b
Building bootstrap
    Finished dev [unoptimized] target(s) in 0.03s
Building stage0 library artifacts (aarch64-apple-darwin)
   Compiling std v0.0.0 (/Users/yukang/rust/library/std)
error[E0432]: unresolved imports `addr2line::LookupContinuation`, `addr2line::LookupResult`
   --> library/std/src/../../backtrace/src/symbolize/gimli.rs:162:25
    |
162 |         use addr2line::{LookupContinuation, LookupResult};
    |                         ^^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^ no `LookupResult` in the root
    |                         |
    |                         no `LookupContinuation` in the root

error[E0412]: cannot find type `SplitDwarfLoad` in crate `addr2line`
   --> library/std/src/../../backtrace/src/symbolize/gimli/macho.rs:330:23
    |
330 |     _load: addr2line::SplitDwarfLoad<EndianSlice<'data, Endian>>,
    |                       ^^^^^^^^^^^^^^ not found in `addr2line`

Some errors have detailed explanations: E0412, E0432.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `std` (lib) due to 2 previous errors
Build completed unsuccessfully in 0:00:01
```

anything need to be fixed?

Reply:
I mean, that `miniz_oxide` should be updated too, to sync version with upstream backtrace.

Reply:
Duped `miniz_oxide` version via `flate2`, maybe touch it too?

Reply:
update `flate2` to `1.0.26`.